engine/src/conversion/apivec.rs

```diff
@@ -97,6 +97,10 @@ impl<P: AnalysisPhase> ApiVec<P> {
         self.apis.iter()
     }
 
+    pub(crate) fn iter_mut(&mut self) -> impl Iterator<Item = &mut Api<P>> {
+        self.apis.iter_mut()
```

Comment:
Oh, I'm really sorry, I offered you bad advice here. `iter_mut` is considered harmful here for the reasons explained in this comment:
https://github.com/google/autocxx/pull/1206/files/8d4d785f4105c3332b1f77f8f0c0756ef9e72545..c90cd896eca376977c87e04707ec48ba01b243bd#diff-dd61892ff96d055fbcab83c5eecc76c8b6120c3eb2f1b5687a8c025435200fcdL32
I just plain forgot! Sorry for the rework.